### ISSUE TITLE
webpack 5 integration doesn't work - "SyntaxError: Invalid or unexpected token"

### ISSUE THREAD

**Issue by mmaietta** [herberttn/bytenode-webpack-plugin#12]
Error:
```
/Development/electron-app/out/main/main.compiled.jsc:1
M��q|s�/�
 ^

SyntaxError: Invalid or unexpected token
```

My apps only output file is `./src/main/index.ts` -> `./out/main/main.js`

The webpack output code of `main.js` is below
```
(() => {
var exports = {};
exports.id = "main";
exports.ids = ["main"];
exports.modules = {

/***/ "./node_modules/@herberttn/bytenode-webpack-plugin/lib/loaders.js?{\"imports\":[\"./src/main/index.ts\"]}!":
/*!*************************************************************************************************************!*\
  !*** ./node_modules/@herberttn/bytenode-webpack-plugin/lib/loaders.js?{"imports":["./src/main/index.ts"]}! ***!
  \*************************************************************************************************************/
/***/ ((__unused_webpack_module, __unused_webpack_exports, __webpack_require__) => {

__webpack_require__(/*! bytenode */ "bytenode");
__webpack_require__(/*! ./src/main/index.ts */ "./src/main/index.ts");

/***/ }),

/***/ "bytenode":
/*!***************************!*\
  !*** external "bytenode" ***!
  \***************************/
/***/ ((module) => {

"use strict";
module.exports = require("bytenode");;

/***/ })

};
;

// load runtime
var __webpack_require__ = require("./main.compiled.jsc");
__webpack_require__.C(exports);
var __webpack_exec__ = (moduleId) => (__webpack_require__(__webpack_require__.s = moduleId))
var __webpack_exports__ = (__webpack_exec__("./node_modules/@herberttn/bytenode-webpack-plugin/lib/loaders.js?{\"imports\":[\"./src/main/index.ts\"]}!"));
var __webpack_export_target__ = exports;
for(var i in __webpack_exports__) __webpack_export_target__[i] = __webpack_exports__[i];
if(__webpack_exports__.__esModule) Object.defineProperty(__webpack_export_target__, "__esModule", { value: true });

})();
```

It's barfing at this line: `var __webpack_require__ = require("./main.compiled.jsc");`

If I set the file content of `main.js` to just be below, it all works perfectly.
```
require("bytenode"); 
require("./main.compiled.jsc");
```

I've been trying to debug how the content is created, but it seems to be failing due to this logic:
https://github.com/herberttn/bytenode-webpack-plugin/blob/e4375bec10c2ae8c6b0728f78cab62450eb2e5ca/src/index.ts#L91-L102
I see that it's simple a string replacement of `main.compiled.js` -> `main.compiled.jsc`. This doesn't work because bytenode hasn't been loaded yet.

That's about as far into debugging it that I've gotten thus far. It'd be great if we could match the entire `__webpack_require__` line, but I don't know if `file` can ever be a filepath as opposed to just a name

**Comment by herberttn** [comment]
@mmaietta can you try again with `v2.1.0`?

**Comment by mmaietta** [comment]
Hi @herberttn I gave it another shot :)

This time I receive:
```
Error: Cannot find module 'strip-shebang'
Require stack:
- /Users/michaelmaietta/Development/uta-shell/node_modules/@herberttn/bytenode-webpack-plugin/lib/sources.js
- /Users/michaelmaietta/Development/uta-shell/node_modules/@herberttn/bytenode-webpack-plugin/lib/index.js
```

If I install `yarn add -D strip-shebang`, I get this:
```
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /Users/michaelmaietta/Development/uta-shell/node_modules/strip-shebang/index.js
require() of ES modules is not supported.
```

**Comment by mmaietta** [comment]
Side note, was it intentional to remove the `examples` directory from the repo in this latest release?

**Comment by herberttn** [comment]
> Side note, was it intentional to remove the `examples` directory from the repo in this latest release?

There never was an `examples` directory 🤔 
In fact I'm working on one, but never committed, yet

**Comment by herberttn** [comment]
About the `strip-shebang`, looks like I've misplaced it as `devDependencies`.
I'll release a patch shortly.

**Comment by mmaietta** [comment]
Whoops. I just noticed the diff with the .gitignore haha. Must have confused it for the [bytnenode examples](https://github.com/bytenode/bytenode/tree/master/examples) then.

And sounds great.

Just tested with electron-builder + webpack and the app works flawlessly :)

**Comment by herberttn** [comment]
[`v2.1.1`](https://github.com/herberttn/bytenode-webpack-plugin/releases/tag/v2.1.1) released

**Comment by herberttn** [comment]
> Just tested with electron-builder + webpack and the app works flawlessly :)

Did you use the default configuration? Had to change anything?
I'll try and an example and CI test for `electron-builder` too

**Comment by mmaietta** [comment]
My work project has a ridiculously complex setup for both webpack and electron-builder. I don't have a quick-start-app setup for it atm. Looks like a template is available though to test with https://github.com/electron-userland/electron-webpack-quick-start
Would love to add a bytenode example to here though: https://www.electron.build/#boilerplates

**Comment by mmaietta** [comment]
Closing this per fix in v2.1.1
